pr_agent/algo/utils.py

```diff
@@ -12,6 +12,7 @@ def convert_to_markdown(output_data: dict) -> str:
     emojis = {
         "Main theme": "🎯",
         "Type of PR": "📌",
+        "Score": "🏅",
```

Comment:
Consider adding a comment explaining what the new 'Score' emoji represents. This will improve code readability. [medium]